src/features/foods/components/BadgeList/index.tsx

```diff
@@ -55,20 +57,25 @@ export const BadgeList = ({
           'flex -mx-4 px-4 gap-1.5 mt-4 pb-1 overflow-x-auto whitespace-nowrap',
       )}
     >
-      {groupName ? (
+      {!!groupName && (
         <FilterBadge
           icon={HomeIcon_Off}
           text={groupName}
-          onCrossClick={() => updateUrlParams('group')}
+          onCrossClick={() => {
+            updateUrlParams('group');
+            updateUrlParams('container');
+          }}
```

Comment:
Added `updateUrlParams('container')` so that when the cross button of the group filter badge is clicked, the container filer badge gets removed together.